packages/components/shared/src/events/toggle.ts

```diff
@@ -2,6 +2,7 @@ export type SlToggleEvent<T = unknown> = CustomEvent<T>;
 
 declare global {
   interface GlobalEventHandlersEventMap {
-    'sl-toggle': SlToggleEvent;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    'sl-toggle': SlToggleEvent<any>;
```

Comment:
I think `any` is ok here, since you can't predict how it's being used. Within components though, you will also use the `SlToggleEvent<T = unknown>` type.

Reply:
ok, so let's leave it then :)